lib/livebook/live_markdown/export.ex

```diff
@@ -70,8 +70,8 @@ defmodule Livebook.LiveMarkdown.Export do
   defp notebook_metadata(notebook) do
     put_unless_default(
       %{},
-      Map.take(notebook, [:persist_outputs, :autosave_interval_s]),
-      Map.take(Notebook.new(), [:persist_outputs, :autosave_interval_s])
+      Map.take(notebook, [:persist_outputs, :autosave_interval_s, :secrets]),
+      Map.take(Notebook.new(), [:persist_outputs, :autosave_interval_s, :secrets])
```

Comment:
We don't want to store secrets in the notebook because they would be saved in plain text.

What we will do is to encrypt notebooks in the future. Then we can store secrets in encrypted format and you can access them as long as you have the key.